docs/source/getting-started/install.rst

```diff
@@ -234,6 +238,45 @@ Un-installation
 
 ----
 
+.. _docker_install:
+
+Installation via Docker: 
+------------------------
+
+You can install Scancode Toolkit by building a Docker image from the included Dockerfile.
+
+Download the ScanCode-Toolkit Source Code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Either download the Source Code for one of the releases ( :ref:`_app_install` ) and unzip it.
+- Or git clone the latest ( :ref:`_source_code_install` ) Source Code.
+
+
+Build the Docker image
+^^^^^^^^^^^^^^^^^^^^^^
+
+The ``docker build`` command needs to run in the directory of the source code, make sure to ``cd`` into the correct directory.
+
+    cd scancode-toolkit
+    docker build -t scancode-toolkit .
+
+Run using Docker
+^^^^^^^^^^^^^^^^
+
+The docker image will forward all arguments it receives directly to the ``scancode`` command. 
+
+Display help:
+
+    docker run scancode-toolkit --help
+
+Mount current working directory and run scan on mounted folder:
+
+    docker run -v $(pwd):/project scancode-toolkit <arguments> /project
```

Comment:
And where do go the output?

Reply:
I'll add a line explaining and expand the sample - any output written to the mounted directory like `/project/my-output-file.json` inside the docker command will create a file `$PWD/my-output-file.json` in the current working directory on the host machine. Alternatively, a second directory could be mounted with a further -v command to create a separate output directory.